Source/Shaders/Builtin/Functions/pbrLighting.glsl

```diff
@@ -0,0 +1,95 @@
+const float M_PI = 3.141592653589793;
+
+vec3 lambertianDiffuse(vec3 diffuseColor) {
+  return diffuseColor / M_PI;
+}
+
+vec3 fresnelSchlick2(vec3 f0, vec3 f90, float VdotH) {
+    return f0 + (f90 - f0) * pow(clamp(1.0 - VdotH, 0.0, 1.0), 5.0);
```

Comment:
Out of curiosity are there any notable differences between the shaders here and the shaders in `processPbrMaterials.js` or are they basically the same?

Reply:
The shaders here are essentially the same as the PBR portions from `processPbrMaterials.js`. I did cross reference with a few other sources to check for differences, but the calculations are as described in the glTF spec, the sample viewer, and [this tutorial](https://learnopengl.com/PBR/Lighting) which was helpful in understanding the math

Reply:
Great!